docs/api/core.md

```diff
@@ -2,7 +2,9 @@ This doc describes core `jotai` bundle.
 
 ## atom
 
-```ts
+`atom` is an object where we save data there. It's immutable and shouldn't get mutated/changed directly.
```

Comment:
we want to clarify the `atom()` creator function, and the created atom config object. Provider holds "atom value" and atom config doesn't hold any values (thus can be used for multiple providers.) We should clarify the concept in `docs/introduction/concepts.md`.

Reply:
```suggestion
`atom` is a function to create an atom config. The atom config is an immutable object. The atom config itself doesn't hold an atom value. The atom value is stored in a Provider state.
```